### DISCUSSION TITLE
add a modchart editor

### DISCUSSION THREAD

**Discussion by stripes10122** [JordanSantiagoYT/FNF-JS-Engine#740] [Ideas]
adding a modchart editor will maek modcharts esier  (i speak engrish)

**Comment by TheStinkern** [comment]
you can't be serious

modchart editor will never be added, go ahead and close this

**Reply by stripes10122** [reply]
how i close it

**Reply by TheStinkern** [reply]
> how i close it

forgot that this is not an issue, I have not made a discussion on a repository

I feel like you are underaged because of the grammar

"how I close it"

**Reply by stripes10122** [reply]
ye
